bootstrap/Dockerfile.Builder

```diff
@@ -18,10 +18,3 @@ RUN mkdir -p $GOPATH/src/github.com/kubeflow/kubeflow/bootstrap
 COPY ./glide* $GOPATH/src/github.com/kubeflow/kubeflow/bootstrap/
 RUN cd $GOPATH/src/github.com/kubeflow/kubeflow/bootstrap/ && \
     glide install -v
-
```

Comment:
So builder only containers vendor, and we only need to rebuild it if glide is changed, right?

Reply:
Yes